xicam/gui/widgets/linearworkfloweditor.py

```diff
@@ -74,8 +80,13 @@ def __init__(self, workflow: Workflow, operation_filter: Callable[[OperationPlug
 
     def run_workflow(self, **kwargs):
         mixed_kwargs = self.kwargs.copy()
+        mixed_kwargs.update(self.kwargs_callable(self))
```

Comment:
what is the kwargs_callable intended for?

Reply:
Now annotated in the docstring:

> A callable that gets called when auto-run is triggered. This callable is expected to generate a dict of
            kwargs that will be passed into the workflow as inputs.